pkg/networkutils/network.go

```diff
@@ -456,23 +493,22 @@ func (n *linuxNetwork) buildIptablesSNATRules(vpcCIDRs []string, primaryAddr *ne
 			"-m", "comment", "--comment", "AWS SNAT CHAIN", "-j", "AWS-SNAT-CHAIN-0",
 		}})
 
-	for i, cidr := range allCIDRs {
-		curChain := chains[i]
-		curName := fmt.Sprintf("[%d] AWS-SNAT-CHAIN", i)
-		nextChain := chains[i+1]
+	curChain := "AWS-SNAT-CHAIN-0"
+	curName := "AWS-SNAT-CHAIN-0"
```

Comment:
nit: these variables are unnecessary when we can just use `chain` defined above. You can rename it to `chainName` if it is easier